### ISSUE TITLE
latest version doesn't works on OS X 10.10.2

### ISSUE THREAD

**Issue by shuge** [dagwieers/unoconv#248]
```
unoconv -h
unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```

unoconv: latest version from git
LibreOffice Version: 4.4.1.2
Python 2.7.9

**Comment by smellman** [comment]
https://github.com/smellman/unoconv/commit/d1405f781216a571b6111915d70ee80a8cf1a877
I just write patch to be run.
It seem LibreOffice file structure was changed.

**Comment by HadrienKulik** [comment]
Thank you! It worked perfectly for me :) https://github.com/dagwieers/unoconv/issues/211

**Comment by retrography** [comment]
This works great, @smellman. Great job!

**Comment by mik01aj** [comment]
I found another workaround. Just install LibreOffice.

**Comment by BeatsMe** [comment]
I've still have the problem.
```
$ brew update
$ brew install --HEAD unoconv
$ unoconv -h
unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```
```
$ git clone https://github.com/dagwieers/unoconv
$ cd unoconv
$ ./unoconv -h
unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```
I've also tried on the environment below.
* LibreOffice_4.4.2_MacOS_x86-64.dmg
* `brew cask install libreoffice`
* OS X Yosemite Version 10.10.2

**Comment by BeatsMe** [comment]
The problem has gone after downgrading to LibreOffice v4.3.6.
The file structure under /Applications/LibreOffice.app/Contents/MacOS between 4.4.2 and 4.3.6 are very different. Needs modification on https://github.com/dagwieers/unoconv/blob/master/unoconv .

**Comment by Kequc** [comment]
I had success with.

    brew cask install libreoffice
    brew install unoconv --HEAD

Seems it's working as such with LibreOffice_4.4.3_MacOS_x86-64

**Comment by skywinder** [comment]
Duplicate of #163. Fixed in #250.

Use `brew install unoconv --HEAD` until new version will not released.
